addons/resource.language.en_gb/resources/strings.po

```diff
@@ -23724,3 +23724,385 @@ msgstr ""
+msgstr ""
+
+#. Remove video version dialog title
+#: xbmc/video/dialogs/GUIDialogVideoVersion.cpp
+msgctxt "#40018"
+msgid "Remove video version"
+msgstr ""
+
+#. Add new video version dialog text
+#: xbmc/video/dialogs/GUIDialogVideoVersion.cpp
+msgctxt "#40019"
+msgid "Unable to remove default version \"{0:s}\" from {1:s} \"{2:s}\". Change the default version and try again."
+msgstr ""
+
+#. Remove video version dialog text
+#: xbmc/video/dialogs/GUIDialogVideoVersion.cpp
+msgctxt "#40020"
+msgid "Are you sure to remove the video version \"{0:s}\"?"
+msgstr ""
+
+#. Convert video version menu
+#: xbmc/video/dialogs/GUIDialogVideoVersion.cpp
+msgctxt "#40021"
+msgid "Convert {0:s} version"
+msgstr ""
+
+#. Manage video version dialog title
+#: xbmc/video/dialogs/GUIDialogVideoVersion.cpp
+msgctxt "#40022"
+msgid "Manage {0:s}"
+msgstr ""
+
+#. Play video version dialog title
+#: xbmc/video/dialogs/GUIDialogVideoVersion.cpp
+msgctxt "#40023"
+msgid "Play {0:s}"
+msgstr ""
+
+#. Play default video version setting
+#: system/settings/settings.xml
+msgctxt "#40200"
+msgid "Play default video version"
+msgstr ""
+
+#. Help for play default video version setting
+#: system/settings/settings.xml
+msgctxt "#40201"
+msgid "Select player action for video with multiple versions.[CR][Enabled] Automatically play the default video version without prompting.[CR][Disabled] Always display a dialogue to select the preferred video version."
+msgstr ""
+
+#. Ignore different video versions settting
+#: system/settings/settings.xml
+msgctxt "#40202"
+msgid "Ignore different video versions on scan"
+msgstr ""
+
+#. Help for Ignore different video versions settting
+#: system/settings/settings.xml
+msgctxt "#40203"
+msgid "Select scanner action for different video versions.[CR][Enabled] Add different video versions to library seperately without confirmation.[CR][Disabled] Prompt for converting different video versions into additional versions of the original."
+msgstr ""
+
+#. Ignore video extras settting
+#: system/settings/settings.xml
+msgctxt "#40204"
+msgid "Ignore video extras on scan"
+msgstr ""
+
+#. Help for ignore video extras settting
+#: system/settings/settings.xml
+msgctxt "#40205"
+msgid "Select scanner action for video extras in \"extras\" folder.[CR][Enabled] Scan video extras like regular videos.[CR][Disabled] Add video extras to library for associated video."
+msgstr ""
+
+#. Show video versions as folder settting
+#: system/settings/settings.xml
+msgctxt "#40206"
+msgid "Show video with multiple versions as folder"
+msgstr ""
+
+#. Help for show video versions as folder settting
+#: system/settings/settings.xml
+msgctxt "#40207"
+msgid "When enabled, video with multiple versions will be shown as folder in library, this folder can then be opened to display the individual video versions. When disabled, a video version dialog will be opened for the video."
+msgstr ""
+
+# Video versions
+
```

Comment:
a couple of labels still miss the contextual description, which is needed for translators to do correct translations. Please add those in a follow up PR one this one is merged. Thanks.
Example for the label below: `#. Name of a video/movie version, like "director's cut"`

Reply:
Yes, let's add the missing descriptions in a followup PR.